src/coreclr/jit/unwindarm64.cpp

```diff
@@ -638,14 +638,26 @@ void Compiler::unwindSaveNext()
 void Compiler::unwindPacSignLR()
 {
 #if defined(FEATURE_CFI_SUPPORT)
-    // do not use unwindSaveNext when generating CFI codes as there is no code for this
-    assert(!generateCFIUnwindCodes());
+    if (generateCFIUnwindCodes())
+    {
+        FuncInfoDsc*   func     = funCurrentFunc();
+        UNATIVE_OFFSET cbProlog = 0;
+        if (compGeneratingProlog)
+        {
+            cbProlog = unwindGetCurrentOffset(func);
+        }
+
+        // DW_CFA_GNU_window_save 0x2D
+        createCfiCode(func, cbProlog, CFI_DEF_CFA_REGISTER, DWARF_REG_ILLEGAL);
```

Comment:
I am not sure how the arguments CFI_DEF_CFA_REGISTER, DWARF_REG_ILLEGAL translate to the DW_CFA_GNU_window_save.

Reply:
It doesn't. This is a temporary thing to unblock other CI jobs. I am trying to figure out how to come up with the encode to pass to `createCfiCode()` . I do not see codes that are being passed to that method.